src/erlcloud_httpc.erl

```diff
@@ -53,8 +53,8 @@ request_httpc(URL, Method, Hdrs, <<>>, Timeout, _Config) ->
                                  [{body_format, binary}]));
 request_httpc(URL, Method, Hdrs, Body, Timeout, _Config) ->
     {value,
-     {_, ContentType}, HdrsRest} = lists:keytake(<<"content-type">>, 1, Hdrs),
-    HdrsStr = [{to_list_string(K), to_list_string(V)} || {K, V} <- HdrsRest],
+     {_, ContentType}, _HdrsRest} = lists:keytake("content-type", 1, Hdrs),
+    HdrsStr = [{to_list_string(K), to_list_string(V)} || {K, V} <- Hdrs],
```

Comment:
In our testing and reviewing the httpc request logic (in OTP 17.4) we did not see the content-type header being added. We ended up switching to httpc and modifying the header logic because we were generating empty content-type headers with lhttpc which caused our backend s3 store to choke. Good catch on using keyfind instead of keytake and searching after any binaries had been converted to strings.